sdks/java/core/src/main/java/org/apache/beam/sdk/testing/BigqueryMatcher.java

```diff
@@ -165,14 +165,16 @@ QueryResponse queryWithRetries(Bigquery bigqueryClient, QueryRequest queryConten
         }
       } catch (IOException e) {
         // ignore and retry
-        LOG.warn("Ignore the error and retry the query.");
+        LOG.warn("Retrying query ({}) after exception", queryContent.getQuery(), e);
```

Comment:
accidentally push my commit... Thanks for the suggestion. I just wanted to ask the best place to log without changing retry loop from do..while() to while()...